CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ The most recent changes are on top, in each type of changes category.
 
 ## [Unreleased]
 
+### Added
+
+- Password change front #158 (https://github.com/cyfronet-fid/sat4envi/issues/158)
```

Comment:
The formatting is still incorrect though

Reply:
Ok, I got it. Case was to link up #158, now it should be fine

Reply:
Now formatting is fine, but the place is wrong again. :D